### PR TITLE
Feature/ae nil return

### PR DESCRIPTION
**Pull request by willdot**
PBI: 71840

### CODE REVIEW

File: src/EA.Weee.Web/Areas/AeReturn/Controllers/ReturnsController.cs [hunk 1]
```diff
@@ -0,0 +1,157 @@
...
+            }
+        }
+
+        [HttpPost]
+        [ValidateAntiForgeryToken]
+        public async Task<ActionResult> ExportedWholeWeee(Guid organisationId, ExportedWholeWeeeViewModel viewModel)
+        {
+            if (!ModelState.IsValid)
+            {
+                await SetBreadcrumb(organisationId, BreadCrumbConstant.AeReturn);
```
Comment (SamGoodeySFW) [inline src/EA.Weee.Web/Areas/AeReturn/Controllers/ReturnsController.cs]: Should this be moved out of the if statement? If there's an issue we'll lose the breadcrumb

File: src/EA.Weee.Web/Areas/AeReturn/Views/Returns/ExportedWholeWeee.cshtml [hunk 2]
```diff
@@ -0,0 +1,66 @@
...
+            }
+        </div>
+    </div>
+</section>
+
+
+@section scripts
+{
+    <script>
+        function showWarning(show) {
```
Comment (SamGoodeySFW) [inline src/EA.Weee.Web/Areas/AeReturn/Views/Returns/ExportedWholeWeee.cshtml]: Could we create a script file for this piece of JS? We can then bundle it and add it to this page

File: src/EA.Weee.Web/Areas/AeReturn/Controllers/ReturnsController.cs [hunk 3]
```diff
@@ -0,0 +1,157 @@
...
+
+                return View(viewModel);
+            }
+        }
+
+        [HttpPost]
+        [ValidateAntiForgeryToken]
+        public ActionResult Index(ReturnsViewModel viewModel)
+        {
+            return AeRedirect.ExportedWholeWeee(viewModel.OrganisationId);
```
Comment (JB12298) [inline src/EA.Weee.Web/Areas/AeReturn/Controllers/ReturnsController.cs]: Is this a placeholder?  Otherwise it needs a unit test

**Comment by JohnPadgettCivica** [comment]
Some merge conflicts and just needs unit test as per Jans comment to check the route redirect

**Pull request merged**